### ISSUE TITLE
SciHub

### ISSUE THREAD

**Issue by Pfern** [Pfern/OSODOS#17]
SciHub may not survive the current war with well funded publishers.
But, in the scope of the 2017 panorama it plays an important role in transforming the methods of access.
Can we converge on a way to refer to it?

**Comment by rvosa** [comment]
I don't think we can refer to it by name, at least not as a recommendation. Maybe there is a news story that we can cite?
